comp/core/ipc/impl-none/ipc.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package noneimpl implements a noop version of the auth_token component
+package noneimpl
```

Comment:
What is the use case for the noneimpl ?

Reply:
I see, ideally the tracer would not depends on `remote-config` in serverless mode. But that's for another PR !